components/folders/TabsSection/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+
+import Box from "@mui/material/Box";
+import Tabs from "@mui/material/Tabs";
+import Tab from "@mui/material/Tab";
+import Typography from "@mui/material/Typography";
+
+const TabsSection = () => {
+  return (
+    <>
+      <Box
+        mb={2}
+        display="flex"
+        justifyContent="center"
+        sx={{ borderBottom: 1, borderColor: "divider" }}
+      >
+        <Tabs
+          value={"value"}
+          onChange={() => console.log("change")}
+          aria-label="basic tabs example"
+        >
+          <Tab label={"Aperçu".toUpperCase()} />
```

Comment:
You can remove those `toUpperCase`, because MUI already uses the CSS rule `text-transform: uppercase;`